docs/src/atoms/promptExample.jsx

```diff
@@ -0,0 +1,26 @@
+import {
+    A,
+    Button,
+    DropdownButton,
+    Prompt,
+} from 'react-cm-ui';
+import React from 'react';
+
+function PrompExample() {
+    return (
+        <div>
+            <Prompt inline>
+                <Button color="success">Save Me!</Button>
+            </Prompt>
+
+            <br />
+            <br />
+
+            <Prompt inline>
+                <A>asdf</A>
```

Comment:
Should this say something other than "asdf"?  "Click Me"?

Reply:
Thanks!